app/assets/stylesheets/views/_homepage.scss

```diff
@@ -159,3 +159,48 @@
   @include govuk-font(36, $weight: bold);
   color: govuk-colour("white");
 }
+
+.homepage-links-and-search {
+  background: govuk-colour("light-grey");
+  padding-bottom: govuk-spacing(8);
+  padding-top: govuk-spacing(6);
+
+  @include govuk-media-query($from: "desktop") {
+    padding-bottom: govuk-spacing(8);
+  }
+}
+
+.homepage-search {
+  @include govuk-media-query($until: "tablet") {
+    max-width: 420px;
```

Comment:
Not a suggestion but a musing that I wonder if this should be specified at the gem level. I can't think of any scenarios that are made better with great big long search bars.

Reply:
So the Frontend has the ability to [set a character width on the text input][1] (seems like a nice use of `max-width`)  which appears to be available on the [gem's input component][2].

Whilst it shouldn't be too difficult to get the search component to use this, I'd suggest that this improvement should be a separate thing done in the component gem - otherwise it'd block this work. Happy to raise this as an issue.

[1]: https://design-system.service.gov.uk/components/text-input/#fixed-width-inputs
[2]: https://components.publishing.service.gov.uk/component-guide/input/with_custom_width

Reply:
Completely agree, a tomorrow problem. I should've been clearer in my verbosity that this is something to think about later. Resolving for now but will make a note to jot up an issue and reference this comment.